lib/generators/solidus_braintree/update/update_generator.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module SolidusBraintree
+  module Generators
+    class UpdateGenerator < Rails::Generators::Base
```

Comment:
Added an UpdateGenerator to add the data migration to the user's app.

Reply:
Do we really need to use a different generator here? I'm thinking that everywhere else we used the install generator for both, relying on Thor's ability to propose diffs for conflicts and ignore identical files.

Thoughts on only using one?

Reply:
Okay, let me see if I can merge it with the InstallGenerator :+1: